tests/common/devices.py

```diff
@@ -39,6 +39,7 @@
 except Exception as e:
     logging.error("Hack for https://github.com/ansible/pytest-ansible/issues/47 failed: {}".format(repr(e)))
 
+logger = logging.getLogger(__name__)
```

Comment:
This one looks strange. But there is logger access in this file without defining it. Not sure how these code were tested, or even tested?